pypdf/_writer.py

```diff
@@ -55,6 +55,7 @@
     Type,
     Union,
     cast,
+    Literal,
```

Comment:
Sadly, we still have to support older versions of Python. For this reason, the import has to be:

```
try:
    from typing import Literal  # type: ignore[attr-defined]
except ImportError:
    # PEP 586 introduced typing.Literal with Python 3.8
    # For older Python versions, the backport typing_extensions is necessary:
    from typing_extensions import Literal  # type: ignore[misc]
```

Reply:
This should be done